### ISSUE TITLE
New EnumExtension switch statement for `ParseX( this string value )` has bad style for `}`

### ISSUE THREAD

**Issue by jaredmoo** [Azure/autorest#2386]
See https://github.com/Azure/azure-sdk-for-net/pull/3391/files/4c5e539dae1f583fff8cac7e19e4af89cd8a3bb8#diff-cc8c43982e0b4b1a7c70e6fe2f307f70R54 .

**Comment by olydis** [comment]
seems like this was not generated with the latest *stable* but some recent nightly - but then again, the glitch you saw fixed as of https://github.com/Azure/autorest/pull/2377, so the most recent nightly already does it right again.

**Comment by jaredmoo** [comment]
Thanks!

**Comment by jaredmoo** [comment]
Should we only be using stable for generating azure-sdk-for-net ?

**Comment by olydis** [comment]
Actually no, sorry if I made that impression, was just overwhelmed by recognizing the bug and thinking "ah but our stable doesn't do that". 
People can use whatever version they like, every nightly is fully tested and such - if they can handle the pace. We started releasing "stables" because some users of autorest understandably prefer consistency over most recent features (e.g. in the validation world - you don't want #errors/warnings to play roller coaster, especially if you want to track your Swagger's quality).
